tests/Unit/RmStepTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+use Symfony\Component\Filesystem\Filesystem;
+use WordPress\Blueprints\BlueprintException;
+use WordPress\Blueprints\Model\DataClass\RmStep;
+use WordPress\Blueprints\Runner\Step\RmStepRunner;
+use WordPress\Blueprints\Runtime\NativePHPRuntime;
+
+
+beforeEach(function() {
+    $this->runtime = new NativePHPRuntime(sys_get_temp_dir());
+
+    $this->step = new RmStepRunner();
+    $this->step->setRuntime($this->runtime);
+
+    $this->fileSystem = new Filesystem();
+});
+
+it('should remove a directory when an absolute path is provided', function() {
+    $unresolvedDirToRemove = "dir";
+    $resolvedDirToRemove = $this->runtime->resolvePath($unresolvedDirToRemove);
+    $this->fileSystem->mkdir($resolvedDirToRemove);
+
+    $input = new RmStep();
+    $input->path = $resolvedDirToRemove;
```

Comment:
The step should accept a relative path, not a resolved absolute path. Does this test pass? If so, is this a bug?

Reply:
Aha, this specifically tests what happens when an absolute path is provided. Let's make that apparent in the code structure, e.g. by collapsing these two lines into one:

```diff
-    $unresolvedDirToRemove = "dir";
-    $resolvedDirToRemove = $this->runtime->resolvePath($unresolvedDirToRemove);
+    $resolvedDirToRemove = $this->runtime->resolvePath("dir");
```

Reply:
Haven't we agreed on it accepting any? Internally resolving unresolved paths, and leaving absolute ones be? Could you describe the ideal behavior you expect from this step?

Reply:
Yes we did, I realized after posting the initial comment. My second comment in this thread contains a suggestion to make the test slightly less confusing.

Reply:
Done.